src/openms/include/OpenMS/ANALYSIS/MAPMATCHING/FeatureMapping.h

```diff
@@ -45,9 +46,19 @@ namespace OpenMS
       {
           public:
 
-          struct FeatureToMs2Indices
+          /// Stores information required for preprocessing
+          class FeatureMappingInfo
```

Comment:
Well, this might not be a comment from a reviewer, but I am still confused why sometimes I use class and other structure.

Reply:
There are no clear rules but in general it is good to use structs for simple records / PODs and switch to class if you have e.g. private or protected members.
See e.g., with some hints when not to use class 
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c-classes-and-class-hierarchies for suggestions how to use them.

Reply:
I have to admit that here it was simply personal preference, this could also be a struct.